examples/todoapp/src/components/IDActionCell.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button } from "@chakra-ui/react";
+import {
+  TableCellComponentProps,
+  useFabrixClient,
+  useFabrixContext,
+} from "@fabrix-framework/fabrix";
+import { useCallback, useMemo, useState } from "react";
+
+export const IDActionCell = {
+  name: "IDActionCell",
+  type: "tableCell",
+  component: (props: TableCellComponentProps) => {
+    const client = useFabrixClient();
+    const context = useFabrixContext();
+    const [isMutating, setMutating] = useState(false);
+    const values = useMemo(() => {
+      const label = props.userProps?.["label"] ?? "Action";
+      const color = props.userProps?.["color"] ?? "blue";
+      const mutation = props.userProps?.["mutation"];
+      if (!mutation) {
+        throw new Error("Mutation is required");
+      }
+
+      return {
+        label,
+        color,
+        mutation,
+      };
+    }, [props]);
+    const markDone = useCallback(async () => {
```

Comment:
```suggestion
    const mutate = useCallback(async () => {
```
This is a mutation that is not limited to markDone.

Reply:
Good catch!